pcsx2/x86/iFPU.cpp

```diff
@@ -941,6 +957,7 @@ void recCVT_W()
 		DOUBLE::recCVT_W();
 		return;
 	}
+	EE::Profiler.EmitOp(eeOpcode::CVTW);
```

Comment:
If we have the following EmitOP() on the top then it'll get calculated twice if ``CHECK_FPU_FULL`` is true as we also have a EmitOP() on recCVT_W on iFPUd.cpp. hence have it below the possible return.